### ISSUE TITLE
3D visualization of planning and flight possible?

### ISSUE THREAD

**Issue by GrahamDyer** [ArduPilot/MissionPlanner#1872]
#### Issue details
![airrails2](https://user-images.githubusercontent.com/3287001/41926606-260a024a-7970-11e8-82e3-2f85c0901201.jpg)

The AirRails GCS has an amazing feature where one can view the flight and do the planning in 3D.

Isn't that wonderful, to visualize both in a 3D environment? Is this possible to implement in MP?

There is a video here: https://www.youtube.com/watch?v=LT1lDytQOi4

#### Version
All

#### Platform
[ X ] All
[  ] AntennaTracker
[  ] Copter
[  ] Plane
[  ] Rover
[  ] Sub

#### Airframe type
All
#### Hardware type
All

#### Logs
N/A

**Comment by GrahamDyer** [comment]
Yes, but that's for post-planning or post-flight outside of Mission Planner. You can also upload a KML to http://doarama.com/ for an amazing after-flight visualization.

Having 3D inside Mission Planner is the next big thing.

**Comment by kikislater** [comment]
After flight, kml is available !
Nice Cesium usage !

**Comment by MyOnlyOnlySunshine** [comment]
Hope to join the ArcGIS 3D map
